datafusion/core/src/physical_plan/aggregates/mod.rs

```diff
@@ -65,13 +66,43 @@ pub enum AggregateMode {
     FinalPartitioned,
 }
 
+/// Represents a GROUPING SET in the physical plan
```

Comment:
👍 
Thank you -- this is well commented and clear to understand

```suggestion
/// Represents `GROUP BY` clause in the plan plan (including the more general GROUPING SET)
```

I was thinking that many people encountering SQL will not be familiar with GROUPING SET but would be very familiar with GROUP BY -- what would you think about calling this structure `PhysicalGroupBy` or something to make the connection to `GROUPING` clearer?